### DISCUSSION TITLE
Edfu small finds ideas

### DISCUSSION THREAD

**Discussion by luizaogs** [arch-kiosk/arch-kiosk-office#1621] [users: ideas for new features]
I'm not sure if this should be a discussion. It's quite possible that it should (I don't think ideas are going in the wiki any longer, are they?). But since I believe it is pretty easy to convert an issue into a discussion I thought I would start here. I'm doing the small finds registering at Edfu and while I have just gotten started and might add to this as the season goes on, I wanted to note things that might be of use/interest before I forget to do so. They have separate databases for small finds and ceramics, and I am only dealing with small finds. 

Anyway, a couple of things that strike me as useful right off the bat (and which have already proven useful to me): the ability to search through all of the small finds ever recorded for the site (not just for a specific locus) by type (here "category of object"), material, and context number when registering new objects. I am not sure if this is something that you guys have already envisioned for analysis, but it seems useful for this stage as well. Other possible filters available for the search here include dating, feature, the date the record was created, and who made the record.

Second, they have checkboxes that can be checked to indicate if a small find already has a photo, a drawing, a facsimile, or an enhanced image (with something like DStretch, for instance). Not sure if that is as necessary here since records are more integrated to begin with, but maybe it is helpful to note it? They do not have checkboxes to indicate that a particular type of record still needs creation -- they just note that in the description box -- but perhaps something to think on, too.

They also have an input field for comparanda, which I am sure is something already in discussion for analysis for Kiosk but that might also be useful on the registration front? Not sure. It seems like something mostly added later to the database by the Edfu team.

**Comment by urapadmin** [comment]
from our agent in Edfu: converted to discussion

**Comment by lbestock** [comment]
Hello Small Finds Registrar from Edfu, our very useful agent!

First the little things. 

For comparanda, do they mean comparanda within the set of objects found at Edfu and in the database? Or is this a way to note the need to look at records of objects say in museum collections? Both have utility, but would be accomplished quite differently. Also one is purely a research need and the other more fieldwork and research combined, organizing the storage of like objects in the same place, for instance.

Checking if images are there with a checkbox. Since our images are integrated I don't see this as useful. 

However, it raises a point that the search across all finds raises and that we have indeed not supported, and that is multi-season workflows. Uronarti has an unusual storage situation: we have none. So we really do most of our processing in the season of excavation, and that is what is mostly supported. Study seasons are something to think about. For checkboxes, to me the relevant thing is whether or not an object NEEDS to be drawn or photographed. That's a separate thing than if it HAS been. If I took a crappy picture last season of this seal impression, noted that when I wanted to publish it in the offseason, how do I mark that I need to make sure to photograph it again the next season? Or a field photo does not replace the need for an object photo. This can already be dealt with using the registration codes, but those are pretty famously only intelligible when one is in the daily grind of registration and do not accomplish the bigger picture planning for and executing a study season sort of goals. We should discuss and think about this. Not in v.1.

Searching the finds and sorting across seasons and without reference to loci or units is of course a need. And for ceramics. But I don't see much point in building it into Filemaker and again think it wants thinking, not in v.1. But that's precisely why this is a good discussion ticket. Keep gathering real world experience of things that are valuable, please.

Um and for Lutz, who is always always looking for examples of what people actually do in analysis. In what circumstance do you search across small finds? As in, WHY are you searching? To accomplish what?

**Comment by luizaogs** [comment]
Oh I didn't think any of that would be relevant for v.1. Ok, in order: 

1. In terms of comparanda, they seem to mean both. I have seen records of figurines with comments in the comparanda field mentioning specific places at the site where similar ones were found, as well as records of figurines with bibliographic/museum information from other sites. So it isn't used for storage at all.

2. No, I agree that checkboxes used to note whether an image (or other record) exists are not very useful here, but I think checkboxes that indicate that something is needed could be. That is in fact what I thought they were for at first, before clarifying with Nadine.

3. In terms of searching for finds, I've found it super helpful to see how other registrars have recorded specific types of objects in the past since I am not familiar with the stuff from here yet. Which might not be a very relevant point on other projects. But they do registration quite differently here -- a lot of the finds I am registering right now, for example, are from 2018 - 2020. In which case I need to search for those contexts to make sure I am not doubling numbers (though the database does tell me if I do that), since I am the one giving the objects special finds numbers myself. Which isn't really relevant to how registration is done in your software, though I wonder how something like that would be done if projects that use the software in the future register finds in this way? In that case I do see it being useful even in the registration stage, and of course agree that it is a necessity for analysis.

**Reply by urapadmin** [reply]
For the time being I only reply to one thing: No, it won't be relevant for v1. There is no plan to do anything about registration for the upcoming projects.

**Comment by luizaogs** [comment]
Another thing they do here that is related to small finds is that they give priority numbers of 1-5 to contexts. 5 is most important (so the ancestor shrine they found a couple of years ago, for instance), while 1 is crappy surface/disturbed stuff. That tells the registrar and ceramicist what to focus on.

**Reply by urapadmin** [reply]
we call that "Dear Registrar" and we use the "!" in status to achieve something along that line

**Comment by luizaogs** [comment]
Laurel and I talked about this yesterday briefly and I realized that I forgot to mention here that the search is accomplished through the "find mode" (which seems to be an FM thing? But I'm not sure). It turns the entire form from data entry into data search, which is why all of those different fields were searchable, but I really only used context and feature numbers, material, and recorder (in case I needed to find my own records).

**Reply by urapadmin** [reply]
yes, the Find Mode is the built-in Filemaker querying UI ... that we don't use. Only useful for rather, urm, flat database structures and a less guided user experience. However, we can search for all kind of things without the find mode just we would have to develop a user interface for the particular case and decide on what exactly to search for. So your examples of what you actually searched for are already helpful on that front.

**Reply by luizaogs** [reply]
Ha no, I didn’t think you would use the find mode :) It wasn’t the friendliest UI.
